src/Hintman/Suggest/TrailingNewline.hs

```diff
@@ -0,0 +1,19 @@
+module Hintman.Suggest.TrailingNewline
+       ( suggestNoTrailingNewline
+       ) where
+
+import Hintman (LineNumber)
+
+import qualified Data.Text as T
+
+
+suggestNoTrailingNewline :: (FilePath, Text) -> Maybe (FilePath, LineNumber, Text)
+suggestNoTrailingNewline (path, body) = (generateResult . reverse . dropWhile isNewline . reverse) splitBody
+    where
```

Comment:
2 space before `where` and 4 spaces before functions under `where` according to our style guide :slightly_smiling_face:

Reply:
Also, empty lines between function signatures